### PR TITLE
difftastic: init at unstable-2021-07-09

### PR DESCRIPTION
**Pull request by ethancedwards8**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 129815` at 109ab654 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>difftastic</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: unused-argument

Unused argument: `stdenv`.
Near pkgs/tools/text/difftastic/default.nix:1:8:
```
  |
1 | { lib, stdenv, fetchFromGitHub, rustPlatform }:
  |        ^
```
</li>
  </ul>
</details>

**Comment by ethancedwards8** [comment]
@r-rmcgibbo eval

File: pkgs/tools/text/difftastic/default.nix [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{ lib, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "difftastic";
+  version = "unstable-2021-07-09";
```
Comment (SuperSandro2000) [inline pkgs/tools/text/difftastic/default.nix:5]:
```suggestion
  version = "0.3";
```

File: pkgs/tools/text/difftastic/default.nix [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+  src = fetchFromGitHub {
+    owner = "wilfred";
+    repo = pname;
+    rev = "1dc6ab3977e4d81e71cff1ac4289987773f64b58";
```
Comment (SuperSandro2000) [inline pkgs/tools/text/difftastic/default.nix:10]:
```suggestion
    rev = version;
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Pull request merged**